### ISSUE TITLE
Final Report Peer Review - eer48

### ISSUE THREAD

**Issue by easonrecto** [fjt37/ml-big-data-project#14]
The goal of this report is to predict the price of a bitcoin given information such as the number of bitcoins in the world, the number of transactions made, the hash rate, and the number of unique IP addresses mining bitcoins on a given day. The team took a three-step approach to creating their model. They first explored their data, searching for features that gave the most direct information. Next, they performed some preliminary analysis with a variety of linear regressions to assess the quantity of each feature. Finally, they fit a myriad of models (switching loss functions, regularizers, and data subsets) to the data and selected the best one according to their chosen error metric.
One thing I liked about this project right off the bat was the deep intuition for the problem setting and awareness of their data expressed during exploratory data analysis. Knowing to drop inaccurate data and know whether two features give the same information is critical to optimizing a model which they did very well. I also liked the resolution of the conflict with outliers that natrually occur with bitcoin. Specifically, acknowledging that they may not want to use a robust loss function was very interesting. In terms of improvement, (though I know space in this report was precious) I think the data could have been either explained a bit clearer or visualized in such a way that made it easier to understand. Also, regarding visuals, I would have liked to see a table of results to see comparisons of the models that were chosen so I can think about how different losses and regularizers affected the predictions. Finally, future work on this question can be tried using unsupervised learning such as a neural network to predict prices. Doing so could allow for different mixes  of or different sequences of the given information to provide better predictions.
Overall, I thought this project was awesome and you guys did a great job! I learned a good amount about bitcoin and may take a look into it more!
